test/cppsuite/test_harness/configuration_settings.h

```diff
@@ -39,22 +44,22 @@ class configuration {
         }
     }
 
-    std::string
+    const std::string &
     get_config()
     {
         return (_config);
     }
 
     /*
      * Wrapper functions for retrieving basic configuration values. Ideally the tests can avoid
-     * using the config item struct provided by wiredtiger. However if they still wish to use it the
-     * get and next functions can be used.
+     * using the config item struct provided by wiredtiger. However if they still wish to use it
+     * the get and next functions can be used.
      */
     int
-    get_string(const char *key, std::string &value)
+    get_string(const std::string &key, std::string &value)
     {
```

Comment:
Function can be const

Reply:
Yes, as discussed, all the following functions will be const:
get_config, get_string, get_bool, get_int, next, get